pkg/scsi/scsi.go

```diff
@@ -454,3 +482,25 @@ func (s *scsi) waitUdevSymlink(ctx context.Context, deviceName string, wwn strin
 	logger.Info(ctx, "udev symlink for %s with WWN %s found", deviceName, wwn)
 	return nil
 }
+
+func (s *scsi) waitUdevSymlinkNVMe(ctx context.Context, deviceName string, wwn string) error {
+	var checkPath string
+	if strings.HasPrefix(deviceName, "dm-") {
+		checkPath = diskByIDDMPathNVMe + wwn
+	} else {
+		checkPath = diskByIDSCSIPath + wwn
+	}
+	symlink, err := s.filePath.EvalSymlinks(checkPath)
+	if err != nil {
+		msg := fmt.Sprintf("symlink for path %s not found: %s", checkPath, err.Error())
+		logger.Info(ctx, msg)
```

Comment:
log as error, 
same comment in line 501

Reply:
changed to error